### PR TITLE
Clean code: Vertical Formatting - Ordering - Issue #68

### PR DESCRIPTION
**Pull request by wintondeshong**
Fixes issue #68 

- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [n/a] Automated tests are passing
- [n/a] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [n/a] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

**Review by dylanjustice — APPROVED** [review]

**Review by jstevenson — APPROVED** [review]

File: clean-code/FORMATTING.md [hunk 1]
```diff
@@ -16,7 +16,55 @@ Unnecessary comments, such as those on well-named variables, creates space that
...
 ### Ordering
 
-https://github.com/AndcultureCode/AndcultureCode/issues/68
+While files should be kept small, we realize that due to varying factors files
+extend well beyond 100 lines. Due to this reality, our team leverages regions to organize our
+different members.
+
+Please refer to the following documentation for regions by language:
+
+-   [CSharp](https://github.com/AndcultureCode/AndcultureCode/blob/main/CODE-CONVENTIONS-CSHARP.md)
```
Comment (HeyKos) [inline clean-code/FORMATTING.md:25]: Praise: Appreciate the link to our existing standards docs.

**Review by HeyKos — COMMENTED** [review]
Looks great Winton! Thanks for putting this together.

**Comment by wintondeshong** [comment]
@all-contributors please add @wintondeshong for maintenance, review and documentation

**Comment by wintondeshong** [comment]
@all-contributors please add @dylanjustice for review and documentation

**Pull request merged**